### PR TITLE
fix: Detecting paths relative to template directory in logging

### LINKED ISSUE #166: Not Windows-compatible code in loggers
**Describe the bug**

In https://github.com/pawamoy/mkdocstrings/blob/master/src/mkdocstrings/loggers.py#L107:

```python
return template.filename.rsplit("/templates/", 1)[1]
```

The forward slashes are not cross-platform :wink:

**Additional context**
We should probably use `os.sep` instead of `/` here.

### PR DESCRIPTION
**Pull request by oprypin**
Fixes #166

### CODE REVIEW

File: src/mkdocstrings/loggers.py [hunk 1]
```diff
@@ -102,9 +105,12 @@ def get_template_path(context: Context) -> str:
     Returns:
         The relative path to the template.
     """
-    template = context.environment.get_template(context.name)
-    if template.filename:
-        return template.filename.rsplit("/templates/", 1)[1]
+    filename = context.environment.get_template(context.name).filename
+    if filename:
+        try:
+            return str(Path(filename).relative_to(base.TEMPLATES_DIR))
```
Comment (pawamoy) [inline src/mkdocstrings/loggers.py:111]: I'm not sure this would work for custom templates, which would exist outside of `base.TEMPLATES_DIR`?

Comment (oprypin) [reply src/mkdocstrings/loggers.py:111]: Yes, so an absolute path will be used for those. It's good because otherwise it'd be ambiguous.

Comment (pawamoy) [reply src/mkdocstrings/loggers.py:111]: Oh, perfect then :smile:

**Comment by pawamoy** [comment]
Thanks again! :tada:

**Pull request merged**